### ISSUE TITLE
Staging Review finding: "Showing X results" text should update as filter and sort are applied

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#87886]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** VA BTSSS - Travel Pay
**Product:** Travel Pay
**Feature:** MVP

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

The interface feedback for the list of claims shows only the number of results displayed, i.e. "Showing 1-10 of 153 results." This text should be updated to include the sort order and whether any filters have been applied.

When reordering or filtering results, it may be difficult for some users to determine what the current status of the page is, and whether the Order or Apply filters buttons have already been pressed or still need to be pressed. A user trying to determine what the status of the page is would have to compare what results are currently displayed against what they remember being there previously (high cognitive load).

There are a few accessibility scenarios where this could be an issue:
- A screen reader user, eg. a blind user who doesn't get the visual cue that something has happened on the page
- A screen magnification/browser zoom user, eg. a low vision user at high magnification (pushing the rest of the content out of the viewport)
- Any user who was distracted from their computer and then returned to it but can't remember whether they clicked the button or not, eg. need to use the bathroom, distracted by a child, received a phone call, etc.

For users with data limitations for their device, pressing a button again has a small but real financial cost. In general, better to give a clear indication of status rather than make them guess or have to re-submit.

## Recommended action

Update the "Showing X results" text dynamically to indicate the sort order and whether filters are applied. Example: "Showing 1-10 claims of 153, sorted by date (newest), with 7 filters applied."  (This also pairs nicely with the focus management recommendations made in another ticket.)

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Low-bandwidth connection
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
